packages/relay/tests/lib/config/hbarSpendingPlanConfigService.spec.ts

```diff
@@ -125,17 +146,19 @@ describe('HbarSpendingPlanConfigService', function () {
 
     describe('populatePreconfiguredSpendingPlans', function () {
       describe('negative scenarios', function () {
-        it('should not throw an error if the configuration file is not found', async function () {
-          sinon.stub(fs, 'existsSync').returns(false);
-          await expect(hbarSpendingPlanConfigService.populatePreconfiguredSpendingPlans()).not.to.be.rejected;
-        });
```

Comment:
Previously reviewed code had missing configuration files throwing errors.  It can be updated to log messages instead:
```
      try {
        const configFilePath = findConfig(spendingPlanConfig);
        if (configFilePath && fs.existsSync(configFilePath)) {
          const fileContent = fs.readFileSync(configFilePath, 'utf-8');
          return JSON.parse(fileContent) as SpendingPlanConfig[];
        } else {
          throw new Error(`Configuration file not found at path "${configFilePath ?? spendingPlanConfig}"`);
        }
      } catch (fileError: any) {
        throw new Error(`File error: ${fileError.message}`);
      }
```